app/views/layouts/_navbar.html.erb

```diff
@@ -15,7 +15,58 @@
         <li class="nav-item">
           <a class="nav-link active" href="#"><%= t ".reviews" %></a>
         </li>
+        <% if controller.controller_name == "home" || controller.controller_name == "tours" %>
+          <li class="nav-item">
+             <a class="nav-link active" data-bs-toggle="modal" data-bs-target="#exampleModal">
+              <%= t ".search" %>
+            </a>
+
+            <div class="modal fade" id="exampleModal" tabindex="-1" aria-labelledby="exampleModalLabel" aria-hidden="true">
+              <div class="modal-dialog">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <h5 class="modal-title" id="exampleModalLabel"><%= t ".find_tour" %></h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                  </div>
+                  <div class="modal-body">
+                    <%= search_form_for(@q, url: tours_path, method: :get) do |f| %>
```

Comment:
@q là gì em, anh có thể thay thế bằng biến khác không em

Reply:
@anlvq-1058 Dạ @q là tên tham số tìm kiếm mặc định của ransack mình hoàn toàn có thể config thành biến khác được anh ạ vd @search ...